.travis.yml

```diff
@@ -3,12 +3,15 @@ sudo: false
 
 language: python
 python:
-  - "2.6"
   - "2.7"
-  - "3.3"
   - "3.4"
   - "3.5"
-
+  - "3.6"
+matrix:
```

Comment:
Why add a matrix specifically for 3.7?

Reply:
because without matrix neither 3.7 nor 3.7.1 does not work
https://github.com/travis-ci/travis-ci/issues/9815
https://github.com/travis-ci/travis-ci/issues/9831